_posts/0000-01-02-muku7um.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Learning commit
```

Comment:
*This text will be italic*
_This will also be italic_

**This text will be bold**
__This will also be bold__